roles/os_setup/tasks/powertools.yml

```diff
@@ -4,7 +4,7 @@
 - name: Enable PowerTools repository
   ansible.builtin.command:
     cmd: dnf config-manager --set-enabled powertools
-  when: ansible_os_family == "RedHat"
+  when: ansible_os_family == "RedHat" and ansible_distribution_major_version < "9"
```

Comment:
I only tried == in this case, but when you know it works than merge :)

Reply:
No idea as well, should we use https://github.com/marketplace/actions/ansible-lint ?

Reply:
I tested it locally by running the following task against a Rocky 9 test instance of mine

playbook:
```
---
---
- name: Test conditionals
  hosts: "{{ host_variable | default('all') }}"
  become: true
  tasks:
    - name: Test conditionals less than or equal to 9
      ansible.builtin.debug:
        msg: "test conditionals less than or equal to 9"
      when: ansible_os_family == "RedHat" and ansible_distribution_major_version <= "9"
```

You can play around with the `<=9` with `<9`, `<=8` and so on. Only for `<=9` the debug message will be printed.

Also, I ran the `ansible-lint` command on this playbook and there were no errors.